src/views/course-detail.vue

```diff
@@ -33,7 +33,6 @@ Dependencies:
         <div class="col">
           <div id="main-content-anchor"></div>
 
-          <!-- v-if="viewPermit()" removed, readd when enrollment is reactivated -->
           <component
             :is="contentToDisplay.name"
             v-if="contentToDisplay"
```

Comment:
should be `viewPermit()`, otherwise users can see courses they are not enrolled in by entering the URL